modules/integration/src/test/scala/scala/cli/integration/ExportTestProjects.scala

```diff
@@ -314,4 +314,14 @@ object ExportTestProjects {
            |}
            |""".stripMargin
     )
+
+  def majorScalaVersionTest(scalaVersionMajor: String): TestInputs =
```

Comment:
The word "major" is confusing here, IMO
```suggestion
  def scalaVersionTest(scalaVersion: String): TestInputs =
```